README.md

```diff
@@ -155,43 +154,45 @@ The following use cases are all motivating to this work and it is our goal to pr
 
 ## Key scenarios
 
-These APIs together enable login and linking scenarios that I have put into three  categories.
+These APIs together enable login and linking scenarios that I have put into a few categories.
+For all of these, imagine that an identity provider would store a credential on the user's browser when they log in.
+
 
 ### Scenario 1: User intends to link to an identity provider they are not logged into
 
-In this case, our user has not used this identity provider (idp.net) on this site (example.com). They first interact with some UI in the page that is clearly associated with the identity provider and the following is called.
+In this case, our user is not even logged into this this identity provider (`idp.net`), just having navigated to this site (`example.com`).
+They first interact with some UI in the page that is clearly associated with the identity provider and the following is called.
 
 ```js
 let credential = await navigator.credentials.get({
   identity : {
     providers : [
       {
         loginURL : "https://login.idp.net/login.html",
+        origin: "https://login.idp.net", // may be omitted, inferred from loginURL
       },
     ]
   }
 });
 ```
 
-Browser UI is shown to the user that lets them pick to link their account to the identity provider. On selection, the browser redirects the navigatable to `https://login.idp.net/login.html`.
-There, the user may do some auth flow and on completion, the identity provider calls the following:
+The browser sees there is no credential in the credential store that would work for `example.com`.
+So it falls back and opens the `loginURL` in a tab of the RP's choosing.
+Since it is not specified, we assume the current tab.
+There, the user goes through some authentication and/or authorization flow entirely of the identity provider's choosing, after which the identity provider stores a credential with some code like this:
 
 ```js
 let cred = await navigator.credentials.create({
   identity : {
-    effectiveQueryURL: "https://api.login.idp.net/v1/foo",
-    uiHint: {
-      name: "example human readable",
-      iconURL: "https://api.login.idp.net/v1/photos/exampleUser",
-    }
```

Comment:
Why no UI hints? 😨

Reply:
They still exist lower down. Just trimming up the example to the core functional bits.